releases/v19.2.0-alpha.20190701.md

```diff
@@ -0,0 +1,155 @@
+- CockroachDB now supports the special case `set timezone = 'utc'` as a special alias for `set timezone = 'UTC'`. The other time zone names are still case-sensitive as previously, pending resolution of issue #36864. [#38195][#38195]
+- Add support for `ALTER TABLE ... ALTER COLUMN ... SET NOT NULL`, which adds a non-null constraint to an existing column. [#37554][#37554]
+- `pg_get_constraintdef` now outputs `CHECK` constraints exactly the same way that Postgres does. [#38318][#38318]
+- The `format_type` Postgres builtin now properly respects its second argument. [#38318][#38318]
+- [`SHOW CREATE TABLE`](../v19.2/show-create.html) now prints foreign keys more like Postgres does. [#38318][#38318]
+- The `pg_type_is_visible` builtin from Postgres is now implemented. [#38318][#38318]
+- Casting an int2vector to a string now produces a Postgres-compatible result. [#38318][#38318]
+
+### Command-line changes
+
+- CockroachDB will now print out an error message and an informative hint if the time zone database is unusable. [#38197][#38197]
+
+### Admin UI changes
+
+- The snapshots graph on the [Replication page](../v19.2/admin-ui-replication-dashboard.html) in the Web UI now includes learner snapshots. [#38147][#38147]
+- Remove version information from login screen. [#38140][#38140]
+- Adding `Implicit Transaction` flag to [Statements](../v19.2/admin-ui-statements-page.html) and [Statement Details](../v19.2/admin-ui-statements-page.html#statement-details-page) page. [#36341][#36341]
+
+### Bug fixes
+
+- The `max_retries` column in the statement statistics now properly displays the maximum number of times a statement was retried. [#38028][#38028]
+- Previously, due to a bug when estimating result set sizes in the optimizer, queries involving int ranges that were very large could result in poor plans being generated. [#38036][#38036]
+- Fix bug where MVCC value at future timestamp is returned after a transaction restart. [#38085][#38085]
+- The YCSB workload will no longer ignore the `--db` option. [#38236][#38236]
+- Localities in the `crdb_internal` tables were displayed JSON objects instead of strings. This format stripped them of their order, which is important in localities. This string format now exactly matches what is used in the command line for setting localities. [#37944][#37944]
+- Fixed an incorrect type mismatch error when empty array values are used as default values (and potentially in other contexts). [#38294][#38294]
+- Fix a panic that could occur when decoding decimals as query parameters. [#38304][#38304]
+- Nulls are now correctly handled by `MIN`, `SUM`, and `AVG` [aggregate functions](../v19.2/functions-and-operators.html#aggregate-functions) when used as [window functions](../v19.2/window-functions.html). [#38110][#38110]
+- Prevent a possible panic during [`JSONB`](../v19.2/jsonb.html) binary query parameter parsing. [#38346][#38346]
```

Comment:
Prevent > Prevented 
(Need to make the change in the next three issues as well)

Reply:
Fixed.